### PR TITLE
Clean up Makefile

### PR DESCRIPTION
**Pull request by dlorenc**
This makes the cross targets more explicit, which will prevent recursive make calls and let builds run in parallel.

### CODE REVIEW

**Comment by dlorenc** [comment]
@minikube-bot retest this please

**Comment by dlorenc** [comment]
@minikube-bot retest this please

File: Makefile [hunk 1]
```diff
@@ -42,23 +42,31 @@ LOCALKUBE_LDFLAGS := "$(K8S_VERSION_LDFLAGS) $(MINIKUBE_LDFLAGS) -s -w -extldfla
 
 MKGOPATH := mkdir -p $(GOPATH)/src/$(ORG) && ln -s -f $(shell pwd) $(GOPATH)/src/$(ORG)
 
-LOCALKUBEFILES := $(shell go list  -f '{{join .Deps "\n"}}' ./cmd/localkube/ | grep k8s.io | xargs go list -f '{{ range $$file := .GoFiles }} {{$$.Dir}}/{{$$file}}{{"\n"}}{{end}}')
-MINIKUBEFILES := $(shell go list  -f '{{join .Deps "\n"}}' ./cmd/minikube/ | grep k8s.io | xargs go list -f '{{ range $$file := .GoFiles }} {{$$.Dir}}/{{$$file}}{{"\n"}}{{end}}')
+LOCALKUBEFILES := go list  -f '{{join .Deps "\n"}}' ./cmd/localkube/ | grep k8s.io | xargs go list -f '{{ range $$file := .GoFiles }} {{$$.Dir}}/{{$$file}}{{"\n"}}{{end}}'
```
Comment (jimmidyson) [inline Makefile]: Why change these?

Comment (dlorenc) [inline Makefile]: It speeds things up a bit by only conditionally evaulating these now. If you run a make clean for example, these commands won't get executed.

Comment (jimmidyson) [inline Makefile]: Makes sense.

**Comment by jimmidyson** [comment]
LGTM

**Pull request merged**